chat-commands.js

```diff
@@ -2450,20 +2447,14 @@ const commands = {
 		if (!(user.can('lock') || localPunished)) return this.errorReply(`User ${name} is neither locked nor muted/banned from this room.`);
 
 		if (targetUser && (cmd === 'hidealtstext' || cmd === 'hidetextalts' || cmd === 'hidealttext')) {
-			this.addModAction(`${name}'s alts' messages were cleared from ${room.title} by ${user.name}.`);
+			room.sendMods(`${name}'s alts' messages were cleared from ${room.title} by ${user.name}.`);
```

Comment:
`sendMods` is definitely not the correct function here.

Ideally, this should be sent:

1. to everyone in the room, not just mods
2. as a mod action

but _not_ added to the scrollback log, because future room joiners shouldn't see anything.

Reply:
What do you mean by "to everyone in the room // as a mod action"?

Reply:
A mod action is a message in the format:

`|c|USERNAME|/log MESSAGE`

It appears like `MESSAGE`, except:

1. It'll show a timestamp if timestamps are on
2. It'll notify like a chat message (strong notification if `MESSAGE` highlights you, weak notification ever)
3. It'll be classified as a message from `USERNAME` (so `USERNAME` will not get notified at all, even if their username appears in `MESSAGE`)

Reply:
It looks like you'll need to manually send that construct; rooms only support `addByUser` and `sendModsByUser`.

Reply:
Do we want ``sendByUser`` as part of rooms, or should I just implement it there?

Reply:
Either is fine.